app/models/repository.rb

```diff
@@ -20,6 +20,11 @@ def github_app_installed?
     app_installation_id.present?
   end
 
+  def commentable?
+    return app_installation.write_issues? && display_subject? unless app_installation.nil?
```

Comment:
I think this logic may need expanding to add in conditions about `FETCH_SUBJECT` and personal access tokens, which will allow commenting even if there's an app installation with only `read_issues`